src/common/buffer.cc

```diff
@@ -1369,7 +1369,7 @@ using namespace ceph;
   uint32_t buffer::list::iterator_impl<is_const>::crc32c(
     size_t length, uint32_t crc)
   {
-    length = MIN( length, get_remaining());
+    length = std::min((size_t)length, (size_t)get_remaining());
```

Comment:
tip: you can avoid casts by specifying which template specialization of std::min() you want:
```c++
length = std::min<size_t>(length, get_remaining());
```